ci/test-readme/Dockerfile.opensuse

```diff
@@ -23,7 +23,8 @@ FROM opensuse/tumbleweed:${DISTRO_VERSION}
 
 # ```bash
 RUN zypper refresh && \
-    zypper install -y cmake gcc gcc-c++ git libcurl-devel libopenssl-devel make
+    zypper install  --allow-downgrade -y cmake gcc gcc-c++ git gzip \
```

Comment:
Nit: It looks like there are two spaces there.

Reply:
Fixed.